src/index.html

```diff
@@ -4,9 +4,283 @@
+        <div class="cards__card card">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="producte image"
+          >
+          <h2 class="card__name-product">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code-producte">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars">
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star"></div>
+            </div>
+            <div class="card__count">Отзывов: 5</div>
+          </div>
+          <div class="card__cost">
+            <span class="card__price-text">Цена:</span>
+            <span class="card__price">69 999 грн</span>
+          </div>
+          <a class="card__link" href="#">купить</a>
+        </div>
+        <div class="cards__card card">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="producte image"
+          >
+          <h2 class="card__name-product">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code-producte">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars">
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star"></div>
+            </div>
+            <div class="card__count">Отзывов: 5</div>
+          </div>
+          <div class="card__cost">
+            <span class="card__price-text">Цена:</span>
+            <span class="card__price">69 999 грн</span>
+          </div>
+          <a class="card__link" href="#">купить</a>
+        </div>
+        <div class="cards__card card">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="producte image"
+          >
+          <h2 class="card__name-product">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code-producte">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars">
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star"></div>
+            </div>
```

Comment:
let's change background with nth-child, not with hardcoded class card__star_active